nawqa-pesticide-nsp.Rproj

```diff
@@ -0,0 +1,13 @@
+Version: 1.0
```

Comment:
This surprises me. I've had other project insist that it stays. Do you have a little more info on why it should come out?

Reply:
From @lindsayplatt:
.Rproj files have some settings based on your local R setup. For example, when you close an R project and then open it the next day, it usually opens up whatever files you had open when you last closed it by default. My understanding is that info is stored in the .Rproj, so we usually avoid committing them because they contain info that is unique to your setup. Everything opens an Rproj locally when they setup a new repo, so the .Rproj files will get created during that process. They aren't needed to recreate the pipeline or use the repo code.

Reply:
Thanks for the clarification. Will uncommit this.

Reply:
This seems inconsistent with other projects, as Julie mentions here. 

Here is what an .Rpoj file looks like
```text
Version: 1.0

RestoreWorkspace: Default
SaveWorkspace: Default
AlwaysSaveHistory: Default

EnableCodeIndexing: Yes
UseSpacesForTab: No
NumSpacesForTab: 2
Encoding: UTF-8

RnwWeave: Sweave
LaTeX: pdfLaTeX
```
it contains some info (such as how tabs work) that should be shared for each of the contributors to the repo. For example, if Julie changes her local .Rproj to `UseSpacesForTab: Yes`, there will be a lot of unnecessary diffs in future commits from Julie vs the others that have kept `No`. The file does not have storage of prior opened files, it just sets preferences (`RestoreWorkspace`).

I've given Julie the opposite advice that she's getting here so we should probably sort this out as a crew.

Reply:
Hmmm I feel like I was taught the opposite so whenever I create a new repo, the first thing I do is add `*.Rproj` to my `.gitignore`. Seems like something that would be good to have someone test/investigate and then make a suggestion for the branch. Happy to change my pattern, but that's not how I've been setting up my repos.

Reply:
I don't have strong feelings about this either way. The RStudio folks say to add it https://community.rstudio.com/t/should-rproj-files-be-added-to-gitignore/1269/3

Reply:
If the RStudio folks say it, then do it we shall!!

Reply:
Ok, next PR it'll be in there